### ISSUE TITLE
代码实现请教

### ISSUE THREAD

**Issue by hadoop78** [desperadoccy/async-FL#12]
本人想实现一个分层联邦学习训练，根据代码解释，group: dict, 用于组间的聚合，
请问组内聚合应该如何实现

**Comment by desperadoccy** [comment]
"updater": {
      "path": "updater.SemiAsyncUpdater.SemiAsyncUpdater",
      "update": {
        "path": "update.FedAT.FedAT",
        "params": {
        }
      },
      "loss": "torch.nn.functional.cross_entropy",
      "params": {
      },
      "group": {
        "path": "update.FedAvg.FedAvg",
        "params": {
        }
      }
    }
即组间使用FedAvg聚合，组内使用FedAT聚合，即只需要实现需要的聚合算法，将它的包路径在配置文件中写明
ps：最新版本可能对分层FL不太适配，建议使用release版本
